supplementary_style_guide/glossary_terms_conventions/general_conventions/g.adoc

```diff
@@ -1,7 +1,7 @@
 [discrete]
 [[gplusplus]]
 ==== image:images/yes.png[yes] G++ (noun)
-*Description*: "G++" is a C++ compiler usually operated using the command line. When referring to the program, use "G++".
```

Comment:
Could be out-of-scope, but I wonder if the "G++" and "g++" entries could be consolidated?

Reply:
Good idea. I've consolidated the two entries